go/kbfs/simplefs/simplefs.go

```diff
@@ -1573,6 +1598,28 @@ func (k *SimpleFS) SimpleFSMove(
 					return err
 				}
 
+				if !arg.OverwriteExistingFiles {
+					// If srcPath is a file, use fs.Create to make a
+					// placeholder to avoid overwriting an existing file. This
+					// doesn't avoid conflicts when journal flushes, but
+					// protects from overwriting over an existing file without
+					// causing conflicts.  We don't need to do it for
+					// directories since libkbfs makes sure we can't rename
+					// over a non-empty directory.
+					fi, err := fs.Stat(srcPath)
+					if err != nil {
+						return err
+					}
+					if !fi.IsDir() {
+						if f, err := fs.OpenFile(destPath,
+							os.O_RDWR|os.O_CREATE|os.O_EXCL, 0600); err != nil {
+							return err
```

Comment:
It looks like `OpenFile` will return a non-standard error in the case of an exclusive create where the file already exists: https://github.com/keybase/client/blob/5c0b8cc1968a061e908ea3f25fa08b9d3b6958ff/go/kbfs/libfs/fs.go#L380-L381

So it might not get translated correctly.  Is that ok, or will it mess up the frontend side?

Reply:
Thanks for the heads up! We are translating the error now (above in the deferred function), and we are getting the cause of the error: https://github.com/keybase/client/pull/22704/files#diff-b9b12d1b48b0bfd42d3ebb71a8bf0ffeR90 so it should be fine. GUI looks for the two error codes added.

Reply:
Oh right sorry, I misread `errors.Wrap` and thought it was `errors.New`, without a proper type that can be checked.  I see it now.